cob_mecanum_controller/src/cob_mecanum_controller/mecanum_controller.cpp

```diff
@@ -0,0 +1,46 @@
+
+/*
+ * Copyright 2020 Fraunhofer Institute for Manufacturing Engineering and Automation (IPA)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cob_mecanum_controller/mecanum_controller.h"
```

Comment:
would `#include <cob_mecanum_controller/mecanum_controller.h>` be better

same in `X_node.cpp`